src/TerraformCommand.groovy

```diff
@@ -1,3 +1,26 @@
-interface TerraformCommand {
-    public String getEnvironment()
+trait TerraformCommand {
+    private String environment
+    private String terraformBinary = "terraform"
+
+    public String getEnvironment() {
+        return environment
+    }
+
+    public void setEnvironment(String environment) {
+        this.environment = environment
+    }
```

Comment:
Not all terraform commands have environments - eg: `TerraformValidate`.  I don't think environment should be part of the interface.

Reply:
I can remove it, but that does add in a bit more copy/paste.

Reply:
Fixed.